src/calculation/HTMLBuilder.ts

```diff
@@ -92,23 +92,50 @@ Handlebars.registerHelper('highlightCoverage', (localId, context) => {
  */
 export function sortStatements(measure: fhir4.Measure, groupId: string, statements: StatementResult[]) {
   const group = measure.group?.find(g => g.id === groupId) || measure.group?.[0];
-  const populationSet = new Set(group?.population?.map(p => p.criteria.expression));
+  const populationOrder = [
```

Comment:
This is the same order as the enum itself, so I think you can do
```suggestion
  const populationOrder = Object.values(PopulationType);
```
but if this throws off other things, feel free to ignore.